torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -1341,5 +1360,11 @@ void FuseGraph(std::shared_ptr<Graph>& graph) {
   }
 }
 
+void CustomFuseGraph(std::shared_ptr<Graph>& graph, std::function<bool(Node*)> fn, Symbol tag) {
+  if (canFuseOnCPU() || canFuseOnGPU()) {
```

Comment:
Why do we even check this if it has no connection to what our builtin fuser can handle?

Reply:
Also, `tag` is called a `kind` everywhere in our codebase.